VaticanBAV/et/BAVet150.xml

```diff
@@ -0,0 +1,248 @@
+                        <decoNote xml:id="d1" type="drawing">
+                          <locus target="#1r"/>
+                          <desc>A drawing of a man wearing a black robe of honor, delineated in black and red.</desc>
+                        </decoNote>
+                      </decoDesc>
+                      
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus from="18v" to="19r"/>
+                            <desc type="GuestText"><title ref="LIT5905FidalaHawarya" type="complete">1 John 1–7</title></desc>
+                            <q xml:lang="gez"/>
+                          </item>
+                          
+                          <item xml:id="a2">
+                            <desc type="GuestText"><title ref="LIT4001Syllab"/> written on the inner front board.</desc>
+                            <q xml:lang="gez"/>
+                          </item>
+                          
+                          <item xml:id="a3">
+                            <locus target="#1r"/>
+                            <desc type="Record">Written on top of the page in purple ink.</desc>
+                            <q xml:lang="gez"><persName ref="PRS14405YabasasanA">የበሰሸን፡ አወሲ፡</persName> ፪ ብር፡ ተደበር<supplied reason="omitted">፡</supplied></q>
+                          </item>
+                          
+                          <item xml:id="a4">
+                            <locus target="#2r"/>
+                            <desc type="Unclear"/>
+                            <q xml:lang="gez"><persName ref="PRS14404AccamY">አቻም፡ የለሽ። 
+                              አስካለ<supplied reason="omitted">፡</supplied> ማር<supplied reason="omitted">ያ</supplied>ም፡</persName></q>
+                          </item>
+                          
+                          <item xml:id="a5">
+                            <locus target="#2r"/>
+                            <desc type="Unclear">Note probably originally mentioning the name of the scribe.</desc>
+                            <q xml:lang="gez"><cit><quote><choice><sic>ምብር፡</sic><corr>ምክር፡</corr></choice> ስናይ፡ 
+                              <sic>ለኩሉ፡</sic> ዘይገብራ፡</quote><ref cRef="betmas:LIT2000Mazmur.Ps110.10">Psalm 110:10</ref></cit> ዘጸሐፌሁ፡ ጀ<gap reason="lost"/></q>
+                          </item>
+                          
+                          <item xml:id="a6">
+                            <locus target="#18v"/>
+                            <desc type="Unclear">Written in the upper margin.</desc>
+                            <q xml:lang="gez">በሰላመ፡ ገብርኤል፡ መልአክ፡ በላዕለ፡ ማ<supplied reason="omitted">ርያም፡</supplied></q>
+                          </item>
+                          
+                          <item xml:id="e1">
+                            <desc>The name of <persName ref="PRS14403WaldaG" role="owner">Walda Giyorgis</persName>, probably the original owner of the 
```

Comment:
type="OwnershipNote"

Reply:
It is not an addition, but a "varia" note which summarizes information on the manuscript, so it doesn't need a type